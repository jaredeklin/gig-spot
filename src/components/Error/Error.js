import React from 'react';

export const Error = () => {
  <h2 className="error">
    Nooooooooo!!!!!!! Something went wrong. Please try again.
  </h2>;
};