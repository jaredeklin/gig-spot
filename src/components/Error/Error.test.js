import React from 'react';
import { shallow } from 'enzyme';
import { Error } from './Error';

describe('Error component', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Error />);

    expect(wrapper).toMatchSnapshot();
  });
});
