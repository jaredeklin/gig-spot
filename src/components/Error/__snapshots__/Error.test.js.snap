// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Nooooooooo!!!!!!! Something went wrong. Please try again.",
      "className": "error",
    },
    "ref": null,
    "rendered": "Nooooooooo!!!!!!! Something went wrong. Please try again.",
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Nooooooooo!!!!!!! Something went wrong. Please try again.",
        "className": "error",
      },
      "ref": null,
      "rendered": "Nooooooooo!!!!!!! Something went wrong. Please try again.",
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
